frontend/public/index.html

```diff
@@ -1,67 +1,35 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta
-      name="viewport"
-      content="width=device-width,initial-scale=1.0,user-scalable=no"
-    />
-    <meta
-      name="keyword"
-      content="우아한테크코스, 프로그래밍, 개발자, 백엔드, 프론트엔드, Backend, Frontend, IOS, Android"
-    />
-    <meta
-      name="description"
-      id="metaDesc"
-      content="우아한테크코스에서 개발자들을 위해 디자인된 강의를 수강해보세요."
-    />
-    <meta property="og:type" content="website" />
-    <meta property="og:title" content="우아한테크코스 지원하기" />
-    <meta
-      property="og:description"
-      id="metaOgDesc"
-      content="우아한테크코스에서 개발자들을 위해 디자인된 강의를 수강해보세요."
-    />
-    <meta
-      property="og:image"
-      content="https://apply.techcourse.woowahan.com/assets/logo_thumbnail_bg.png"
-    />
-    <meta property="og:url" content="https://apply.techcourse.woowahan.com/" />
-    <meta name="twitter:card" content="website" />
-    <meta name="twitter:title" content="우아한테크코스 지원하기" />
-    <meta
-      name="twitter:description"
-      id="metaTwitterDesc"
-      content="우아한테크코스에서 개발자들을 위해 디자인된 강의를 수강해보세요."
-    />
-    <meta
-      name="twitter:image"
-      content="https://apply.techcourse.woowahan.com/assets/logo_thumbnail_bg.png"
-    />
-    <meta
-      name="twitter:domain"
-      content="https://apply.techcourse.woowahan.com/"
-    />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link
-      rel="stylesheet"
-      href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
-    />
-    <link
-      rel="stylesheet"
-      href="//cdn.jsdelivr.net/gh/lykmapipo/themify-icons@0.1.2/css/themify-icons.css"
-    />
-    <title>우아한테크코스 지원하기</title>
-  </head>
-  <body>
-    <noscript>
-      <strong
-        >We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work
+<html lang="ko">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no" />
+  <meta name="keyword" content="우아한테크코스, 프로그래밍, 개발자, 백엔드, 프론트엔드, Backend, Frontend, IOS, Android" />
+  <meta name="description" id="metaDesc" content="우아한테크코스에서 개발자들을 위해 디자인된 강의를 수강해보세요." />
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="우아한테크코스 지원하기" />
+  <meta property="og:description" id="metaOgDesc" content="우아한테크코스에서 개발자들을 위해 디자인된 강의를 수강해보세요." />
+  <meta property="og:image" content="https://apply.techcourse.woowahan.com/assets/logo_thumbnail_bg.png" />
```

Comment:
'apply.techcourse.woowahan.com'에서 'apply.techcourse.co.kr'로 변경되었습니다.

```suggestion
  <meta property="og:image" content="https://apply.techcourse.co.kr/assets/logo_thumbnail_bg.png" />
```